src/routes/vote.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+
+const voteController = require('../controllers/vote');
+
+const router = express.Router();
+
+router.get('/register', (req, res) => {
```

Comment:
I think this should be a different path. Because we only want to refresh the timeout, not necessarily vote.
```suggestion
router.get('/voteskip', (req, res) => {
```

Reply:
The ideia behind it is that you are under /vote/register so you are registring your vote. To not be confused with /device/register in which we are registring our devices